src/wp-admin/includes/class-wp-automatic-updater.php

```diff
@@ -1011,26 +1011,90 @@ protected function send_plugin_theme_email( $type, $successful_updates, $failed_
 			$body[] = "\n";
 			$body[] = __( 'Please check your site now. It’s possible that everything is working. If there are updates available, you should update.' );
 			$body[] = "\n";
-
+            
+            // Check for failed themes and plugins listing with failed_update_plugins_themes option_name
+			$failed_update_plugins_themes = (array) get_site_option( 'failed_update_plugins_themes', array() );
```

Comment:
This should be moved outside of this conditional. When a plugin succeeds, it needs to be available. Moving to where the other variables are set up on lines 944-948 should work!

Reply:
Also, let's rename the variable `$failed_update_plugins_themes` to `$past_failures` for simplicity. Also, let's change the option to describe what's stored better. How about `failed_plugin_update_emails`?

Reply:
Yes, made this changes and works good.